lib-multisrc/gigaviewer/src/eu/kanade/tachiyomi/multisrc/gigaviewer/GigaViewer.kt

```diff
@@ -315,12 +383,13 @@ abstract class GigaViewer(
     }
 
     private fun String.toDate(): Long {
-        return runCatching { DATE_PARSER.parse(this)?.time }
+        return runCatching { (if (isPaginated) DATE_PARSER_COMPLEX else DATE_PARSER_SIMPLE).parse(this)?.time }
             .getOrNull() ?: 0L
     }
 
     companion object {
-        private val DATE_PARSER by lazy { SimpleDateFormat("yyyy/MM/dd", Locale.ENGLISH) }
+        private val DATE_PARSER_SIMPLE by lazy { SimpleDateFormat("yyyy/MM/dd", Locale.ENGLISH) }
+        private val DATE_PARSER_COMPLEX by lazy { SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'", Locale.ENGLISH) }
```

Comment:
Move this to the DTO file (at top-level) and remove the lazy (top-level elements are automatically lazy loaded).